### PR TITLE
Pkgs reorg

### PR DESCRIPTION
**Pull request by alessio**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/.idea/
```
Comment (jgimeno) [inline .gitignore:1]: This is the root no? I dont think you expected this but .idea

Comment (alessio) [reply .gitignore:1]: both work

**Review by ilgooz — APPROVED** [review]

**Review by jgimeno — APPROVED** [review]

**Comment by jgimeno** [comment]
I messed it with the conflicts.

**Comment by jgimeno** [comment]
https://github.com/tendermint/cosmos-rosetta-gateway/pull/50

**Pull request closed without merging**